components/resources/brave_components_strings.grd

```diff
@@ -373,9 +373,10 @@
       <!-- WebUI webcompat reporter resources -->
       <message name="IDS_BRAVE_WEBCOMPATREPORTER_REPORT_MODAL_TITLE" desc="Title for broken website report dialog window">Report a broken site</message>
       <message name="IDS_BRAVE_WEBCOMPATREPORTER_REPORT_EXPLANATION" desc="Label for a website's URL indicating that it will be reported for compatibility issues">Let Brave's developers know that this site doesn't work properly with Shields:</message>
-      <message name="IDS_BRAVE_WEBCOMPATREPORTER_REPORT_DISCLAIMER" desc="Describes what information is sent to Brave's servers in a broken website report">Note: This site address will be submitted with your Brave version number and your IP address (which will not be stored).</message>
```

Comment:
I think we added the IP address disclaimer because reports were sent directly to our server, without any kind of IP-stripping proxy in between. Unless that's changed, I'm not sure we should remove it. cc @pes10k

Reply:
Makes sense, will go ahead and put it back